<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XRegistrationFormComponent</class>
 <widget class="QWidget" name="XRegistrationFormComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="person_box">
     <property name="title">
      <string>Person</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="person_label1">
          <property name="text">
           <string>firstname:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="person_name"/>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="person_btn_clear_name">
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="person_label2">
          <property name="text">
           <string>surname:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="person_surname"/>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="person_btn_clear_surname">
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="person_label3">
          <property name="text">
           <string>nickname:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="person_nickname"/>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="person_btn_clear_nickname">
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="person_label4">
          <property name="text">
           <string>password:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="person_password">
          <property name="echoMode">
           <enum>QLineEdit::PasswordEchoOnEdit</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="person_btn_clear_password">
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="person_house">
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>house</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="person_house_female">
           <property name="text">
            <string>female</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="person_house_male">
           <property name="text">
            <string>male</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="person_house_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>162</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="person_label_7">
           <property name="text">
            <string>age:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="person_age">
           <property name="minimum">
            <number>6</number>
           </property>
           <property name="maximum">
            <number>150</number>
           </property>
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>person_name</tabstop>
  <tabstop>person_btn_clear_name</tabstop>
  <tabstop>person_surname</tabstop>
  <tabstop>person_btn_clear_surname</tabstop>
  <tabstop>person_nickname</tabstop>
  <tabstop>person_btn_clear_nickname</tabstop>
  <tabstop>person_password</tabstop>
  <tabstop>person_btn_clear_password</tabstop>
  <tabstop>person_house</tabstop>
  <tabstop>person_house_female</tabstop>
  <tabstop>person_house_male</tabstop>
  <tabstop>person_age</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>person_btn_clear_name</sender>
   <signal>clicked()</signal>
   <receiver>person_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>person_btn_clear_surname</sender>
   <signal>clicked()</signal>
   <receiver>person_surname</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>person_btn_clear_nickname</sender>
   <signal>clicked()</signal>
   <receiver>person_nickname</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>person_btn_clear_password</sender>
   <signal>clicked()</signal>
   <receiver>person_password</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
